### PR TITLE
gubener: Add SNMP to Gubener-Sama link

### PR DESCRIPTION
**Pull request by robertfoss**

### CODE REVIEW

**Comment by robertfoss** [comment]
I would like the record to show that I have no idea what I am doing. Please send help.

File: group_vars/location_gubener/snmp.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+
+snmp_devices:
+  - hostname: gubener-sama-5ghz
```
Comment (spolack) [inline group_vars/location_gubener/snmp.yml:4]: The current hostname is `gubener-sama`, according to assignment ¹ and monitoring ².


[1] https://github.com/Freifunk-Spalter/bbb-configs/blob/81429c151f192651867b2cfc2f75153b1f40a6fb/group_vars/location_gubener/networks.yml#L30

[2] http://monitor.berlin.freifunk.net/host.php?h=sama-sued-5ghz&p=snmp

File: group_vars/location_gubener/snmp.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    address: 10.31.77.160
```
Comment (spolack) [inline group_vars/location_gubener/snmp.yml:5]: The address is the ip of the device to query. According to `networks.yml` it is the second ip in 10.31.77.160/28.

Mesh internal DNS lookup resolves to .162 

```
spolack@spolack:~$ host gubener-sama.olsr
gubener-sama.olsr has address 10.31.77.162
```

File: group_vars/location_gubener/snmp.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    snmp_profile: airos_8
```
Comment (spolack) [inline group_vars/location_gubener/snmp.yml:6]: Furthermore you'd need to make sure that SNMP with community `public` is enabled on the device. The next step would be to build an image for gubener-core and then flash it using `sysupgrade $IMAGE`

Best Simon

**Review by spolack — CHANGES_REQUESTED** [review]

**Comment by PolynomialDivision** [comment]
@robertfoss Here is an example:
image

**Comment by PolynomialDivision** [comment]
@robertfoss If you setup everything correct you have some graphs looking [like this](http://monitor.berlin.freifunk.net/host.php?h=sama-west-5ghz&p=snmp). That is very useful for you and for us. ;) It helps to see whats wrong, or what we can do better and so on.

Just as some explanation. We currently use `collectd` with push services to collect statistics (yeah I know collectd is a bit "old" ;)) The ubiquiti devices allow to querry information about snr, bitrate, ... via the [snmp protocl](https://en.wikipedia.org/wiki/Simple_Network_Management_Protocol). We use collectd on the core-router to query those ubiquti devices and send that to the monitoring servers. So after updating `snmp.yml` you have to reflash the core router, since the core router collects all those statistics from ubiquiti devices.

File: group_vars/location_gubener/snmp.yml (hunk 3 above)
Comment (PolynomialDivision) [reply group_vars/location_gubener/snmp.yml:6]: I have put you an example picture into the comments how to enable it. :)

File: group_vars/location_gubener/snmp.yml (hunk 2 above)
Comment (robertfoss) [reply group_vars/location_gubener/snmp.yml]: Thank you, resolved below.

**Comment by robertfoss** [comment]
After running `$ ansible-playbook play.yml --tags image --limit "gubener-*"` I'm seeing some sysupgrade warnings through the web interface.

@PolynomialDivision Is the image I'm running built using a avm4040 configuration that hasn't been pushed to this repo?

```
Sat Sep 25 23:51:25 CEST 2021 upgrade: Device avm,fritzbox-7530 not supported by this image
Sat Sep 25 23:51:25 CEST 2021 upgrade: Supported devices: avm,fritzbox-4040 Image check failed. 
```

**Comment by robertfoss** [comment]
Flashed the resulting image, and it doesn't seem to explode. Not sure how long I should wait before the snmp stats are reported.

http://monitor.berlin.freifunk.net/host.php?h=gubener-core

**Comment by spolack** [comment]
> Flashed the resulting image, and it doesn't seem to explode. Not sure how long I should wait before the snmp stats are reported.
> 
> http://monitor.berlin.freifunk.net/host.php?h=gubener-core

Find the stats under the hostname of the device.

http://monitor.berlin.freifunk.net/host.php?h=gubener-sama&p=snmp

**Comment by spolack** [comment]
LGTM

**Pull request merged**